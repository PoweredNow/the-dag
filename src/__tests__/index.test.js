const All = require('../');

describe('the-dag packaged correctly ', () => {
  test('exports TheDAG and TheDAGFactory', () => {
    expect(All).toMatchSnapshot();
  });
});
