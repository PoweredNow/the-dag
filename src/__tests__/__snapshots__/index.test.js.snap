// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the-dag packaged correctly  exports TheDAG and TheDAGFactory 1`] = `
Object {
  "TheDAG": [Function],
  "TheDAGFactory": Object {
    "DAGs": Object {},
    "createDAG": [Function],
    "deleteDAG": [Function],
  },
}
`;
